ivadomed/prepare_dataset.py

```diff
@@ -0,0 +1,215 @@
+import nibabel as nib
+import numpy as np
+import matplotlib.pyplot as plt
+import PIL
+import skimage
+import os
+import sys
+from utils import reorient_image
+
+# normalize Image
+def normalize(arr):
```

Comment:
we probably have that code somewhere else

Reply:
@lrouhier many of my comments are unanswered-- so i don't know if you have implemented the feature or if you missed my comments-- the usual thing to do is to link to the commit where the requested changed is made, or to answer something if the requested change is rejected/requires further discussion

Reply:
Yes sorry. It was done in: 306221d